packages/toolshed/src/core/attestations.ts

```diff
@@ -0,0 +1,44 @@
+import { ethers, Wallet } from "ethers"
+
+import { IDisputeManager } from "@graphprotocol/subgraph-service"
+
+/**
+ * Creates an attestation data for a given request and response CIDs.
+ * @param disputeManager The dispute manager contract instance.
+ * @param signer The allocation ID that will be signing the attestation.
+ * @param requestCID The request CID.
+ * @param responseCID The response CID.
+ * @param subgraphDeploymentId The subgraph deployment ID.
+ * @returns The attestation data.
+ */
+export async function createAttestationData(
+  disputeManager: IDisputeManager,
+  signer: Wallet,
```

Comment:
Let's try to use `HardhatEthersSigner` for any wallet/signer type. It's already what we use in most of the codebase and being consistent will lead to less issue.s